src/i18n/en.json

```diff
@@ -122,6 +122,9 @@
       "message-text-default": "Please recheck the form before confirmation",
       "button-text-ok": "Confirm",
       "button-text-cancel": "Cancel"
+    },
+    "pagination": {
+      "btn-label-more": "More"
```

Comment:
`button-text-more`

also let's move it to `common` or smth

Reply:
Fixed

Reply:
button-text-more, not label